src/Type/ObjectType/RootQuery.php

```diff
@@ -366,9 +372,10 @@ public static function register_type() {
 						'description' => __( 'A Theme object', 'wp-graphql' ),
 						'args'        => [
 							'id' => [
-								'type' => [
+								'type'        => [
 									'non_null' => 'ID',
 								],
+								'description' => __( 'xThe globally unique identifier of the theme.', 'wp-graphql' ),
```

Comment:
extra x at the beginning of the description

Reply:
You found my debugging. 😁